### ISSUE TITLE
Patch and create permissions needed in RBAC to read CRD

### ISSUE THREAD

**Issue by javixeneize** [nolar/kopf#923]
### Long story short

Hi

I have built an application to read a custom CRD i have already created. This works perfectly in my local minikube cluster, but when installing it as a docker image in my AWS account (EKS) i am having some problems with RBAC

My code is as simple as this:

```
@kopf.on.create('mycrd')
def create_fn(body, **kwargs):
	print (body)
```

The first problem i am having is that it requires to have **patch** and **create** permissions, which i dont understand why those are needed as i just want to read a resource, i am not writing any new resource or modifying anything. 

I have added those permissions to my rbac file and the 403 error disappears, but i have another problem there...

When the resource is created, my application doesnt do anything, it doesnt print anything in the console. It says it starts to watch the resource, and suddenly.... it stops. I have this in the logs:

```
[2022-05-18 15:58:04,909] kopf._cogs.clients.w [DEBUG   ] Starting the watch-stream for customresourcedefinitions.v1.apiextensions.k8s.io cluster-wide.
[2022-05-18 15:58:04,911] kopf._cogs.clients.w [DEBUG   ] Stopping the watch-stream for customresourcedefinitions.v1.apiextensions.k8s.io cluster-wide.
```

 However, if i restart the pod, it detects the resource, but it says it is unchanged (thats the expected behaviour as it is on.create and nothing new has been created. 

So in short, two questions

1 - Why patch and create permissions are needed to read a CRD
2 - Why it starts to read it and suddenly it stops 

Thanks!

### Kopf version

1.35.4

### Kubernetes version

1.22

### Python version

3.9

### Code

```python
@kopf.on.create('mycrd')
def create_fn(body, **kwargs):
	print (body)
```

**Comment by nolar** [comment]
> 1 - Why patch and create permissions are needed to read a CRD

PATCH permissions are needed because all state-tracking handlers (on-create/update/delete/resume) store the state on the resource's annotations. If you implement your own storage (Postgres, MySQL, whatnot), that will not be needed either.

If you only want to see the resource, try `@kopf.on.event`.

The CREATE permission is not needed. Can you please show the error about the lack of permissions?

> 2 - Why it starts to read it and suddenly it stops

Sorry, I cannot answer this question without a sufficient amount of logs. These 2 lines are not enough. Can you please attach more logs from `kopf run --verbose …`?

**Comment by javixeneize** [comment]
Thanks. The second issue is solved now
